src/PhpGenerator/PhpFile.php

```diff
@@ -125,7 +125,7 @@ public function addTrait($name)
 	public function addNamespace($name)
 	{
 		if (!isset($this->namespaces[$name])) {
-			$this->namespaces[$name] = new PhpNamespace($name);
+			$this->namespaces[$name] = new PhpNamespace($name ?: '');
 		}
```

Comment:
Yes, however it needs more changes. Should I try to prepare it in this pull request or prepare rather another one?